docs/en/tutorial/tutorial_sql_1.md

```diff
@@ -289,23 +302,23 @@ window w30d as (PARTITION BY uid ORDER BY trans_time ROWS_RANGE BETWEEN 30d PREC
 
 **Frequency Statistics**
 
-We make frequency statistics for a given column. 
+We make frequency statistics for a given column as we may need to know the type of the highest frequency, the proportion of the type with the largest number, etc., in each category.
```

Comment:
ratio might be better than proportion

Reply:
the ratio of men to women in this department  stands at 4:6.

the proportion of women in this department is 60%.

so `ratio` maybe more suitable if it is compared with the type with the second largest number?

Reply:
i see, then proportion is fine